docs/usage/examples.md

```diff
@@ -14,31 +14,35 @@ You can find even more usage examples under [`tests/`](https://github.com/cathay
     --8<-- "docs/usage/scripts/10_flight_list.py:script0"
     ```
 
-=== "`fl.data.df`"
+=== "`data.df`"
     
     ```
     --8<-- "docs/usage/scripts/10_flight_list.py:df0"
     ```
 
 #### Paginate all pages
 Queries for next page as long as user doesn't enter `x`, or if there are no pages left.
```

Comment:
Maybe I have to read everything in context, but in the diff, it's hard to make sense of it.

Reply:
More generally, do you have the doc in this branch rendered somewhere?

Reply:
Sure, just configured cloudflare pages for all branches: https://c9384447.flightradar24.pages.dev/